gradio/networking.py

```diff
@@ -32,7 +32,7 @@
 INITIAL_PORT_VALUE = int(os.getenv(
     'GRADIO_SERVER_PORT', "7860"))  # The http server will try to open on port 7860. If not available, 7861, 7862, etc.
 TRY_NUM_PORTS = int(os.getenv(
-    'GRADIO_NUM_PORTS', "100"))  # Number of ports to try before giving up and throwing an exception.
+    'GRADIO_NUM_PORTS', "1000"))  # Number of ports to try before giving up and throwing an exception.
```

Comment:
Is there a reason this is so high? Seems unnecessarily exhaustive. In my opinion the gradio launch should not hang for that long (unless that process is actually faster than I think), and should quit after a reasonable number of ports and request a defined `server_port`

Reply:
Sure